src/shared-lib.ts

```diff
@@ -0,0 +1,41 @@
+import type {ParsedLockFile} from 'lockparse';
```

Comment:
this file will go away before i merge the PR

as it should become some shared library the action also uses